### ISSUE TITLE
Installing using Xampp

### ISSUE THREAD

**Issue by buckl97** [tkrebs/ep3-bs#184]
Hi there,

This software looks great in the demo but I am having difficulty installing it on my machine. I have pretty basic knowledge of composer and ZendFramework. I am using a Xampp package and I have installed composer and ZendFramework. I am wondering if I should download the EP3 version on the website or clone the GitHub version into by htdocs folder. A step-by-step guide for full installation would be great :)

**Comment by buckl97** [comment]
Is is possible to install this system using a Windows operating system and XAMPP?

**Comment by tkrebs** [comment]
Yes, it is possible and actually pretty simple, too. However, some basic knowledge of webservers and databases is required, which is beyond the scope of the booking system documentation. If you are not familiar with Composer, I recommend downloading the package from the website.

**Comment by buckl97** [comment]
I have downloaded the package from the website into my htdocs folder and within a folder named ep3. I have created an empty database called ep3 and I have renamed the local.php file and entered database credentials (database= "ep3" , username= "root" , password=  " "). I am using windows OS so Im wondering if I need to setup the directory permissions? If not, how do I start the setuptool for the database?
